library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -111,85 +73,6 @@ class Client() {
         fun register(codec: ContentCodec<*>) {
             codecRegistry.register(codec = codec)
         }
-
-        /**
-         * Use the {@param api} to fetch any stored keys belonging to {@param address}.
-         *
-         * The user will need to be prompted to sign to decrypt each bundle.
-         */
-        suspend fun authCheck(api: ApiClient, address: String): List<EncryptedPrivateKeyBundle> {
-            val topic = Topic.userPrivateStoreKeyBundle(toChecksumAddress(address))
-            val res = api.queryTopic(topic)
-            return res.envelopesList.mapNotNull {
-                try {
-                    EncryptedPrivateKeyBundle.parseFrom(it.message)
-                } catch (e: Exception) {
-                    Log.e(TAG, "discarding malformed private key bundle: ${e.message}", e)
-                    null
-                }
-            }
-        }
-
-        /**
-         * Use the {@param api} to save the {@param encryptedKeys} for {@param address}.
-         *
-         * The {@param keys} are used to authorize the publish request.
-         */
-        suspend fun authSave(
-            api: ApiClient,
-            v1Key: PrivateKeyBundleV1,
-            encryptedKeys: EncryptedPrivateKeyBundle,
-        ) {
-            val authorizedIdentity = AuthorizedIdentity(v1Key)
-            authorizedIdentity.address = v1Key.walletAddress
-            val authToken = authorizedIdentity.createAuthToken()
-            api.setAuthToken(authToken)
-            api.publish(
-                envelopes = listOf(
-                    EnvelopeBuilder.buildFromTopic(
-                        topic = Topic.userPrivateStoreKeyBundle(v1Key.walletAddress),
-                        timestamp = Date(),
-                        message = encryptedKeys.toByteArray(),
-                    ),
-                ),
-            )
-        }
-
-        suspend fun canMessage(peerAddress: String, options: ClientOptions? = null): Boolean {
```

Comment:
Maybe it would be helpful for people to still be able to use this to see if someone is on V2 if canMessageV3 returns false? They can't do much with that information tho since they can't actually message or create in V2.